Contest Estimator dies when loading class

**Description**

If `.class` in tested project references class which is not present in either project or provided dependencies `.jars`  - when loading it for executing contest estimator throws exception and dies indefinitely.

**To Reproduce**

JDK8, Contest Estimator, `fastjson-1.2.50`. 

**Expected behavior**

Contest Estimator works clean.

**Actual behavior**

An error is thrown and Contest Estimator hangs indefinitely: 
```
java.lang.NoClassDefFoundError: org/springframework/http/converter/GenericHttpMessageConverter
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_352]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_352]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_352]
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473) ~[?:1.8.0_352]
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74) ~[?:1.8.0_352]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369) ~[?:1.8.0_352]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363) ~[?:1.8.0_352]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_352]
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362) ~[?:1.8.0_352]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_352]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_352]
	at org.utbot.contest.ContestEstimatorKt.runEstimator(ContestEstimator.kt:451) [main/:?]
	at org.utbot.contest.ContestEstimatorKt.main(ContestEstimator.kt:342) [main/:?]
Caused by: java.lang.ClassNotFoundException: org.springframework.http.converter.GenericHttpMessageConverter
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387) ~[?:1.8.0_352]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_352]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_352]
	... 13 more
```

**Additional context**

Before this exception warning is shown:
```
14:08:10.439 | WARN  | Smoke test failed for class: com.alibaba.fastjson.support.spring.FastJsonHttpMessageConverter
```	
This may be related to #1700 - may be not all spring is bundled in `.jar`.

--- a/utbot-junit-contest/src/main/kotlin/org/utbot/contest/ContestEstimator.kt
+++ b/utbot-junit-contest/src/main/kotlin/org/utbot/contest/ContestEstimator.kt
@@ -414,7 +414,7 @@ fun runEstimator(
             try {
                 project.classloader.loadClass(fqn).kotlin
             } catch (e: Throwable) {
-                logger.info { "Smoke test failed for class: $fqn" }
+                logger.warn(e) { "Smoke test failed for class: $fqn" }
             }
         }
 
@@ -445,16 +445,30 @@ fun runEstimator(
                         break@outer
                     }
 
-                    val cut =
-                        ClassUnderTest(
-                            project.classloader.loadClass(classFqn).id,
-                            project.outputTestSrcFolder,
-                            project.unzippedDir
+                    try {
+                        val cut =
+                            ClassUnderTest(
+                                project.classloader.loadClass(classFqn).id,
+                                project.outputTestSrcFolder,
+                                project.unzippedDir
+                            )
+
+                        logger.info { "------------- [${project.name}] ---->--- [$classIndex:$classFqn] ---------------------" }
+
+                        tool.run(
+                            project,
+                            cut,
+                            timeLimit,
+                            fuzzingRatio,
+                            methodNameFilter,
+                            statsForProject,
+                            compiledTestDir,
+                            classFqn
                         )
-
-                    logger.info { "------------- [${project.name}] ---->--- [$classIndex:$classFqn] ---------------------" }
-
-                    tool.run(project, cut, timeLimit, fuzzingRatio, methodNameFilter, statsForProject, compiledTestDir, classFqn)
+                    }
+                    catch (e: Throwable) {
+                        logger.warn(e) { "===================== ERROR IN [${project.name}] FOR [$classIndex:$classFqn] ============" }
+                    }
                 }
             }
         }

